src/components/LocationModal.js

```diff
@@ -0,0 +1,34 @@
+import {
+  StyledModal,
+  StyledAddress,
+  StyledClose,
+  StyledButton,
+} from "./styles";
+
+export default function LocationModal({ onClose, show, data }) {
+  if (!show) {
+    return null;
+  }
+  return (
+    <StyledModal aria-label="information about location">
+      <StyledAddress>
+        {data.street} <br />
+        {`${data.city} 
+        ${data.postal_code}`}
+        <br />
+        {data.country}
+      </StyledAddress>
+      <StyledButton
+        type="button"
+        aria-label="close modal button"
+        onClick={onClose}
```

Comment:
```suggestion
        onClick={handleOnClose}
```
could this be renamed?

Reply:
I would only add that if this is a handler it should be `handle..` and not just what it's handling, and of course watch out for where this came from that it's the same naming of course :)